web/test/testgrid-grid-cell.test.ts

```diff
@@ -0,0 +1,46 @@
+import {
```

Comment:
IMO, for this and column-header, header-row, row-header and row tests cases should be integrated into the `testgrid-grid.test.ts` below to holistically test the grid.

this will lead to less code and cleaner file structure while still reliably testing all of the necessary subcomponents.
Filing an issue is fine :)